po/sv.po

```diff
@@ -410,79 +412,76 @@ msgstr "Autentisering krävs för att låsa eller låsa upp aktiva sessioner."
 #: ../src/login/org.freedesktop.login1.policy.in.h:53
 msgid "Allow indication to the firmware to boot to setup interface"
 msgstr ""
+"Tillåt indikering till firmware att starta upp i inställnings-gränssnitt"
```

Comment:
Generally we try to stay away from dashes, so `inställningsgränssnitt` like you use below is the preferred variant.